ReactViewControl.Avalonia/ReactViewRender.Avalonia.cs

```diff
@@ -26,7 +29,7 @@ partial class ReactViewRender : Control {
         }
 
         partial void PreloadWebView() {
-            var window = Host?.FindLogicalAncestorOfType<WindowBase>() ?? GetHiddenWindow();
+            var window = Host?.FindLogicalAncestorOfType<Window>() ?? GetHiddenWindow();
```

Comment:
leave the WindowBase here... WindowBase is the base class so no need to change this one

Reply:
reverted